.github/scripts/check-react-native-log.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+
+echo "cd build-system-tests/mega-apps/${MEGA_APP_NAME}"
+cd build-system-tests/mega-apps/${MEGA_APP_NAME}
+
+# Define the time of seconds to wait the command
+if [ "$PLATFORM" = "ios" ]; then
+  TIME_TO_WAIT=300
+else
+  TIME_TO_WAIT=200
+fi
+
+# Define the color codes
+BLUE_BOLD="\033[1;36m"
+GREEN_BOLD="\033[1;32m"
+RED_BOLD="\033[1;31m"
+YELLOW_BOLD="\033[1;33m"
+RESET="\033[0m"
+
+# Sleep for the specified time
+echo -e "${BLUE_BOLD}Sleep for $TIME_TO_WAIT second...${RESET}"
+sleep $TIME_TO_WAIT
+
+# Check if the log file only has one line
+if [[ $(wc -l <"$LOG_FILE") -eq 1 && $(head -n 1 "$LOG_FILE") == "info Starting logkitty" ]]; then
```

Comment:
What is "logkitty" and what makes it pertinent when reading through the logs?

Reply:
Oh, that's just from `npx react-native log-android`

Reply:
Can we add a comment for this for future readers?

Reply:
Done https://github.com/aws-amplify/amplify-ui/commit/353b3061c2ee6e5b0001313aa91dcf362ac5bc00#diff-7b98d772dcf34e39aa594b77212597ab85e1588897447a5e6cfebdc29eb50287R13-R17